include/simdutf/implementation.h

```diff
@@ -6,6 +6,9 @@
 #endif
 #include <vector>
 #include <tuple>
+#if __cplusplus >= 202002L
```

Comment:
That might fail under Visual Studio... See

https://github.com/simdutf/simdutf/pull/559

Reply:
msvc strikes again! will rework and use the simdutf macros.